### PR TITLE
Add parentheses to arrow functions single parameters (part 4)

### PR DESCRIPTION
**Pull request by teoli2003**
See #18677

### CODE REVIEW

File: files/en-us/web/api/performanceservertiming/index.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ function requestHandler(request, response) {
   };
   response.writeHead(200, headers);
   response.write('');
-  return setTimeout(_ => {
+  return setTimeout((_) => {
```
Comment (Josh-Cena) [inline files/en-us/web/api/performanceservertiming/index.md:49]:
```suggestion
  return setTimeout(() => {
```

File: files/en-us/web/api/presentation_api/index.md [hunk 2]
```diff
@@ -152,11 +152,11 @@ Example codes below highlight the usage of main features of the Presentation API
   const reconnectBtn = document.querySelector("#reconnectBtn");
   const disconnectBtn = document.querySelector("#disconnectBtn");
 
-  stopBtn.onclick = _ => {
+  stopBtn.onclick = (_) => {
```
Comment (Josh-Cena) [inline files/en-us/web/api/presentation_api/index.md:155]:
```suggestion
  stopBtn.onclick = () => {
```

Same as below

File: files/en-us/web/api/readablestreambyobreader/index.md [hunk 3]
```diff
@@ -108,7 +108,7 @@ When the stream is cancelled the controller will in turn call `cancel()` on the
 The example code in [Using readable byte streams](/en-US/docs/Web/API/Streams_API/Using_readable_byte_streams#examples) calls the cancel method when a button is pressed, as shown:
 
 ```js
-button.addEventListener('click', () => { reader.cancel("user choice").then( () => { console.log(`cancel complete`) }) } );
+button.addEventListener('click', () => { reader.cancel("user choice").then() => { console.log(`cancel complete`) }) } );
```
Comment (Josh-Cena) [inline files/en-us/web/api/readablestreambyobreader/index.md:111]:
```suggestion
button.addEventListener('click', () => { reader.cancel("user choice").then(() => { console.log(`cancel complete`) }) } );
```

File: files/en-us/web/api/request/cache/index.md [hunk 4]
```diff
@@ -100,7 +100,7 @@ fetch("some.json", {cache: "only-if-cached", mode: "same-origin", signal: contro
     return res
   })
   .then(function(response) { /* consume the (possibly stale) response */ })
```
Comment (Josh-Cena) [inline files/en-us/web/api/request/cache/index.md:102]:
```suggestion
  .then((response) => { /* consume the (possibly stale) response */ })
```

File: files/en-us/web/api/resizeobserversize/blocksize/index.md [hunk 5]
```diff
@@ -25,8 +25,8 @@ A decimal representing the block size in pixels.
 In this example we return an array of sizing information with {{domxref("ResizeObserverEntry.contentBoxSize")}}. The `blockSize` property returns the block dimension size of the observed element.
 
 ```js
-const resizeObserver = new ResizeObserver(entries => {
-  for (let entry of entries) {
+const resizeObserver = new ResizeObserver((entries) => {
+  for (const entry of entries) {
     let elemSize = entry.contentBoxSize[0];
```
Comment (Josh-Cena) [inline files/en-us/web/api/resizeobserversize/blocksize/index.md:30]:
```suggestion
    const elemSize = entry.contentBoxSize[0];
```

File: files/en-us/web/api/screenorientation/lock/index.md [hunk 6]
```diff
@@ -100,11 +100,11 @@ rotate_btn.addEventListener('click', () => {
 
   const oppositeOrientation = screen.orientation.type.startsWith("portrait") ? "landscape" : "portrait";
   screen.orientation.lock(oppositeOrientation)
-    .then( () => {
+    .then(() => {
       log.textContent = `Locked to ${oppositeOrientation}\n`
       }
     )
-    .catch ( error => {
+    .catch ((error) => {
```
Comment (lionralfs) [inline files/en-us/web/api/screenorientation/lock/index.md:107]:
```suggestion
    .catch((error) => {
```

**Review by Josh-Cena — APPROVED** [review]

**Pull request merged**